<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AUTO ART/CRF</string>
  </property>
  <property name="whatsThis">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#212121;</string>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>500</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>500</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="topMargin">
       <number>30</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <property name="bottomMargin">
       <number>30</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
    font-weight: 700;  
    padding-left: 30px; 
	color:#fd4a24      
}</string>
          </property>
          <property name="text">
           <string>AUTO ART v2.0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_login">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"> border: 1px solid #fd4a24;
border-radius:10px;
padding-left:6px;
color:#fff     </string>
        </property>
        <property name="placeholderText">
         <string>LOGIN</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_senha">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"> border: 1px solid #fd4a24;
border-radius:10px;
padding-left:6px;
color:#fff     </string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
        <property name="placeholderText">
         <string>SENHA</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>40</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true"> border: 1px solid #fd4a24;
border-radius:10px;
color:#fff     </string>
          </property>
          <property name="text">
           <string>Escolher Arquivo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>100000</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color:#fd4a24; background-color:transparent;</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>45</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QRadioButton" name="radioButtonSE">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>40</height>
           </size>
          </property>
          <property name="tabletTracking">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">color:#fff;
border: 1px solid #fd4a24;
border-radius:10px;
padding: 10px;

</string>
          </property>
          <property name="text">
           <string>CFT</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonMA">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 1px solid #fd4a24;
border-radius:10px;
padding: 10px;
color:#fff;</string>
          </property>
          <property name="text">
           <string>MA</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonCE">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 1px solid #fd4a24;
border-radius:10px;
padding: 10px;
color:#fff;</string>
          </property>
          <property name="text">
           <string>CE</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"> border: 1px solid #fd4a24;
border-radius:10px;
padding-left:6px;
color:#fff     </string>
        </property>
        <property name="placeholderText">
         <string>NÃºmero da ART</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_cadastrar">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background: qlineargradient(
        spread: pad, x1: 0, y1: 0.5, x2: 1, y2: 0.5,
        stop: 0 rgba(253, 74, 36, 1),
        stop: 0.44 rgba(255, 126, 0, 1),
        stop: 1 rgba(225, 73, 0, 1)
    );
    color: white;
    border-radius: 10px;
    font-size: 16px;
	font-weight: 700;
}
QPushButton:hover {
    background: qlineargradient(
        spread: pad, x1: 0, y1: 0.5, x2: 1, y2: 0.5,
        stop: 0 rgba(253, 84, 46, 1),
        stop: 0.44 rgba(255, 136, 10, 1),
        stop: 1 rgba(235, 83, 10, 1)
    );
}
QPushButton:pressed {
    background: qlineargradient(
        spread: pad, x1: 0, y1: 0.5, x2: 1, y2: 0.5,
        stop: 0 rgba(243, 64, 26, 1),
        stop: 0.44 rgba(245, 116, 0, 1),
        stop: 1 rgba(215, 63, 0, 1)
    );
}
</string>
        </property>
        <property name="text">
         <string>CADASTRAR</string>
        </property>
        <property name="iconSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
